### ISSUE TITLE
UNSTABLE trunk / before-test / llm-retrieval

### ISSUE THREAD

**Issue by kit1980** [pytorch/pytorch#129718]
The jobs fail with Unexpected HTTP response: 429 (Too Many Requests) when downloading miniconda: https://hud.pytorch.org/pytorch/pytorch/commit/67416a2996349a2339328cac0f7e54c7d3b3c1d9#26785521099

**Comment by malfet** [comment]
Hmm, wouldn't that just mark all tests unstable (as llm-retrieval depends on target-determination and all tests depend on target-determination)?

**Comment by kit1980** [comment]
> wouldn't that just mark all tests unstable

I don't think so. llm-retrieval is unstable already on HUD, but not everything.

**Comment by malfet** [comment]
Closing as job was removed by https://github.com/pytorch/pytorch/pull/129722

**Comment by clee2000** [comment]
The llm-retrieval job is being reenabled and now uses a cache in https://github.com/pytorch/pytorch/pull/129720 to prevent rate limiting in the future.
